precipitation/ngsild-payloads/withdrawal.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "urn:ngsi-ld:WaterWithdrawal:OPR0000144308",
+    "type": "WaterWithdrawal",
+    "withdrawal": {
+      "type": "Property",
+      "value": 0.0,
+      "observedAt": "2020-01-01T00:00:00Z",
+      "observedBy": {
```

Comment:
I have the feeling that the withdrawal is done at some place (the structure), but is not really observed by the structure. Maybe it could be just a relationship from the water withdrawal to the structure?